### PR TITLE
OBSDOCS-880: Fix style issues in DT/OTEL docs

### PR DESCRIPTION
**Pull request by max-cx**
Version(s):

4.16, 4.15, 4.14, 4.13, 4.12

Issue:

https://issues.redhat.com/browse/OBSDOCS-880

Link to docs preview:

- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-updating
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-removing

- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-installing
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-updating
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-removing

- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-removing
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuration-of-otel-collector
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuration-of-instrumentation
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-troubleshooting
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-migrating
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-updating
- https://73697--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-removing

QE review:
- N/A

Additional information:

### CODE REVIEW

File: observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="distr-tracing-jaeger-configuring"]
-= Configuring and deploying the distributed tracing platform Jaeger
+= Configuring
```
Comment (snarayan-redhat) [inline observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-configuring.adoc:3]: I would be inclined to keep this as is since it would be good for user to stay informed about the page they have landed in. Corresponding changes in topic maps file look good to me since it keeps the panel neat however, the page headings should be descriptive enough for better user experience as well as from SEO standpoint.
The same comment applies to all the modified headings.

**Review by snarayan-redhat — COMMENTED** [review]
I have a comment that applies to all the headings. Rest LGTM

File: modules/distr-tracing-features.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ////
 This module included in the following assemblies:
 -service_mesh/v2x/ossm-architecture.adoc
--dist_tracing_arch/distr-tracing-architecture.adoc
+-observability/distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc
```
Comment (skrthomas) [inline modules/distr-tracing-features.adoc:4]: Globally, the module file metadata needs a *, not a - in the list: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

File: observability/otel/otel-updating.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="dist-tracing-otel-updating"]
-= Updating the {OTELShortName}
+= Upgrading
```
Comment (skrthomas) [inline observability/otel/otel-updating.adoc:3]: Global comment, but jw why did you remove the other part of this? In the mod docs guide, I read that title should be a gerund phrase: https://redhat-documentation.github.io/modular-docs/#assembly-guidelines

**Review by skrthomas — COMMENTED** [review]
Apologies to hold merge, but I have a comment and a question. Thank you!!! I know this is a lot of work!!

File: modules/distr-tracing-features.adoc (hunk 2 above)
Comment (max-cx) [reply modules/distr-tracing-features.adoc:4]: @skrthomas, yes, thank you, I am aware of this, and most of the files in this PR already comply with this rule. However, there has been a number of old files that I had to move recently, and these were initially drafted by a developer, hence the different metadata format. These files are going away in a few months because the component they document has already been deprecated and is waiting for removal. In any case, I updated the metadata lines in the following files as you pointed out:
```
        modified:   modules/distr-tracing-accessing-jaeger-console.adoc
        modified:   modules/distr-tracing-architecture.adoc
        modified:   modules/distr-tracing-config-default.adoc
        modified:   modules/distr-tracing-config-ingester.adoc
        modified:   modules/distr-tracing-config-jaeger-collector.adoc
        modified:   modules/distr-tracing-config-query.adoc
        modified:   modules/distr-tracing-config-sampling.adoc
        modified:   modules/distr-tracing-config-storage.adoc
        modified:   modules/distr-tracing-deploy-default.adoc
        modified:   modules/distr-tracing-deploy-production-es.adoc
        modified:   modules/distr-tracing-deploy-streaming.adoc
        modified:   modules/distr-tracing-deployment-best-practices.adoc
        modified:   modules/distr-tracing-features.adoc
        modified:   modules/distr-tracing-install-elasticsearch.adoc
        modified:   modules/distr-tracing-install-jaeger-operator.adoc
        modified:   modules/distr-tracing-install-overview.adoc
        modified:   modules/distr-tracing-removing-instance-cli.adoc
        modified:   modules/distr-tracing-removing-instance.adoc
        modified:   modules/distr-tracing-sidecar-automatic.adoc
        modified:   modules/distr-tracing-sidecar-manual.adoc
        modified:   modules/distr-tracing-upgrading-es5-es6.adoc
```

File: observability/otel/otel-updating.adoc (hunk 3 above)
Comment (max-cx) [reply observability/otel/otel-updating.adoc:3]: @skrthomas, I previously received peer reviewer feedback suggesting that I shorten the headings for a number of reasons: (1) make headings uniform between the page and the nav entry, (2) reduce fluff (these product name attributes expand into long multiple-word product names), and 3) because there is already context and inline mentions that identify the product/component that the page is about, so it's not necessary to add the product/component name to every heading.

Comment (skrthomas) [reply observability/otel/otel-updating.adoc:3]: Thanks for clarifying this. Apologies for providing contradictory peer review. I noticed the context..since you already altered the titles due to a peer review suggestion, you can disregard. This is maybe something peer reviewers should sync on.

File: modules/distr-tracing-features.adoc (hunk 2 above)
Comment (skrthomas) [reply modules/distr-tracing-features.adoc]: Thanks for updating these and the clarifications. Apologies to ask you to update files that are going to be removed soon. I can appreciate why you didn't update them in this case. Sorry...its hard to know these things as a peer reviewer :) For future, if you have things like this that are known diversions for a reason, its helpful to leave a review comment in the PR at the place where the diversion occurs, that way we don't burden you with unnecessary feedback.
